src/modules/core/apollo/apolloClient.ts

```diff
@@ -0,0 +1,100 @@
+import { useMemo } from 'react';
+import {
+  ApolloClient, HttpLink, InMemoryCache, ApolloLink, NormalizedCacheObject, FieldPolicy,
+} from '@apollo/client';
+import merge from 'deepmerge';
+import isEqual from 'lodash/isEqual';
```

Comment:
Don't import all of lodash, use `lodash.isequal` instead because AFAIK it doesn't tree shake properly